Documentation/doc/scripts/test_doxygen_versions.sh

```diff
@@ -94,7 +94,7 @@ echo "done."
 #add post-processing
 cd ./build_doc
 echo "Adding postprocessing..."
-make -j$NB_CORES doc_with_postprocessing  &>> ./build_logs
+make -j$NB_CORES doc_with_postprocessing  & >> ./build_logs
```

Comment:
@lrineau this change is not correct, right?

Reply:
Indeed. The redirection `&>> ./build_logs` is equivalent to `>> ./build_logs 2>&1` (cerr and cout both redirected to the file).

`make -j$NB_CORES doc_with_postprocessing  & >> ./build_logs` launches `make` in the background (without any redirection), and `>> ./build_logs` is a redirection for a non existent command. (Maybe you do not know, but we can write the redirections *before* the command: `> log echo Hello` will write "Hello" in the file `log`.).